resources/views/livewire/event.blade.php

```diff
@@ -0,0 +1,5 @@
+<div>
+    <button wire:click="vote(1)" class="{{$userVote === 1 ? 'text-green-700 hover:text-green-600 focus:text-green-600' :'text-blue-800 hover:text-blue-600 focus:text-blue-600'}}"><i class="fas fa-arrow-circle-up cursor-pointer"></i></button>
```

Comment:
add `focus:outline-none` to this. Also, I think it would be better to replace the up arrow icon with a green check mark when you've upvoted.